pkg/pf/tests/internal/cross-tests/configure.go

```diff
@@ -204,15 +205,9 @@ resource "` + providerName + `_res" "res" {}
 		)
 		contract.Ignore(test.Preview(t)) // Assert that the preview succeeded, but not the result.
 		contract.Ignore(test.Up(t))      // Assert that the update succeeded, but not the result.
-	})
+	}
 
-	skipCompare := t.Failed() || t.Skipped()
-	t.Run("compare", func(t *testing.T) {
```

Comment:
Why did we decide to remove this logic?

Reply:
By collapsing all of this into 1 test, we get this for free from `require.*` and `t.SkipNow`.